src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -233,9 +258,33 @@ protected function doClear($namespace)
             foreach ($this->redis->_hosts() as $host) {
                 $hosts[] = $this->redis->_instance($host);
             }
-        } elseif ($this->redis instanceof \RedisCluster) {
-            return false;
+        } elseif ($this->redis instanceof \RedisCluster || $this->redis instanceof RedisClusterProxy) {
+            $keys = array();
+
+            foreach ($this->redis->_masters() as $nodeParams) {
```

Comment:
Coincidently, I did also fix doClear in #28269, with a slightly different implementation, see
https://github.com/symfony/symfony/pull/28269/files#diff-a87375907f78e0dcbee340d314e3d052L236
For this reason, this should be reverted from this PR.

Reply:
Ok, i've reverted it.